tests/semantic/range/range.dl

```diff
@@ -0,0 +1,30 @@
+.decl a_f, b_f, c_f, d_f, e_f, f_f (x: float)
```

Comment:
Add the standard comment block at the start to explain what the test is for. I think it's worth adding some more comments to explain what each part is for.

Reply:
done